frontend/src/lib/components/DBSchemaExplorer.svelte

```diff
@@ -9,30 +9,25 @@
 	import ObjectViewer from './propertyPicker/ObjectViewer.svelte'
 
 	export let resourceType: string | undefined
-	export let pg: String | undefined = undefined
+	export let resourcePath: String | undefined = undefined
 
 	let drawer: Drawer
 
-	async function getSchema() {
-		const content = `
-import { Client } from "https://deno.land/x/postgres/mod.ts";
-
-export async function main(pg: Postgresql) {
+	const content = {
+		postgresql: `import { Client } from "https://deno.land/x/postgres/mod.ts";
```

Comment:
we should use a static version of postgres instead of latest

Reply:
It might have been a mistake i've done in other places,let's fix it everywhere

Reply:
did not find any other cases